src/Zynga/Framework/Cache/V2/Config/Mock/InMemory/PgData/ProductionTest.hh

```diff
@@ -0,0 +1,43 @@
+<?hh //strict
+
+namespace Zynga\Framework\Cache\V2\Config\Mock\InMemory\PgData;
+
+use
+  Zynga\Framework\Cache\V2\Config\Mock\InMemory\PgData\Production as ConfigUnderTest
+;
+use Zynga\Framework\Cache\V2\Exceptions\InvalidObjectForKeyCreationException;
+use
+  Zynga\Framework\StorableObject\V1\Test\Mock\Valid as ValidExampleObjectRequiredFields
+;
+use
+  Zynga\Framework\StorableObject\V1\Test\Mock\ValidNoRequired as ValidExampleObject
+;
+use Zynga\Framework\Testing\TestCase\V2\Base as TestCase;
+
+class ProductionTest extends TestCase {
+
+  public function createConfigUnderTest(): ConfigUnderTest {
+    return new ConfigUnderTest();
+  }
+
+  public function testGetServerPairings(): void {
+    $config = $this->createConfigUnderTest();
+    $servers = $config->getServerPairings();
+    $this->assertEquals(0, $servers->keys()->count());
+    $this->assertEquals('InMemory', $config->getDriver());
+    $this->assertEquals(3600, $config->getTTL());
+  }
+
+  public function testCreateKeyFromStorableObject_ExceptionWired(): void {
+    $obj = new ValidExampleObjectRequiredFields();
```

Comment:
Why is this name starting with Valid? It seems in context of the test the obj itself will be invalid due to missing requiredFields.

Reply:
I didn't create that class so I can't really speak to the naming. This is testing that an exception is thrown when exporting without all required fields set. These production and staging tests are a straight copy of the existing DevTest.